RecoHGCal/Configuration/python/RecoHGCal_EventContent_cff.py

```diff
@@ -21,6 +21,10 @@
       'keep *_ticlTrackstersMIP_*_*',
       'keep *_ticlTrackstersMerge_*_*',
       'keep *_ticlCandidateFromTracksters_*_*',
+      'keep *_hfnticlTrackstersEM_*_*',
+      'keep *_hfnticlTrackstersMIP_*_*',
+      'keep *_hfnticlTrackstersMerge_*_*',
+      'keep *_hfnticlCandidateFromTracksters_*_*',
```

Comment:
this naming is not most straightforward, I think it would be better to have HFNose as a suffix/postfix rather than a prefix.

Reply:
done